util/monoclock.c

```diff
@@ -94,3 +94,35 @@ monoclock_get_cputime(struct timeval * tv)
 	/* Failure! */
 	return (-1);
 }
+
+/**
+ * monoclock_getres(resd):
+ * Store the timer resolution in ${resd}. If CLOCK_MONOTONIC is available, use
+ * that clock; if CLOCK_MONOTONIC is unavailable, use CLOCK_REALTIME (if
+ * available) or gettimeofday(2).
+ */
+int
```

Comment:
Why?  The way you've written this function, it can't fail.

... perhaps it should be able to fail, e.g., if `clock_getres(CLOCK_MONOTONIC)` fails with something other than `ENOSYS` or `EINVAL`, mind you.